### PR TITLE
Web pod guide

### PR DESCRIPTION
**Pull request by rjkerrison**
### Web Pod guide

[Airtable ticket](https://airtable.com/tbl7CISHKkr8Kdeh2/viwlDIU6cp8cbQYhj/recWkY9CvKKf8di4Q?blocks=hide)

Lego podracing

### Acceptance Criteria

- [ ] Web pod guide created with next steps
- [ ] `src` folder created to tidy up `medicines/web`
- [ ] `yarn dev` works

### Pre-flight Checklist

- [ ] Tests
- [ ] DUXD review approval
- [ ] Accessibility Reviewed
- [ ] Product owner demo

### Pre-merge Checklist

- [ ] Branch test approved

### CODE REVIEW

File: medicines/web/infrastructure/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+Instructions on getting the website set up.
+As steps are completed, the developer completing those steps is responsible for updating this document.
+This document should become a guide for setting up the website infrastructure on a new environment and for connecting to the existing environment.
+
+## Steps
+
+1. Docker time!
+
+   Create a `Dockerfile` in `/medicines/web` which will run `yarn dev`, serving up the Nextjs website.
```
Comment (m-doughty) [inline medicines/web/infrastructure/README.md]: Where do I get this from?

File: medicines/web/infrastructure/README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+   Verify that this is working locally by running the docker image, and connecting to the website.
+
+   You will need to push the docker image to the container registry for each of our environments.
+
+2. Create Kubernetes pod
+
+   First, you'll need YAML specifying the pod's configuration.
```
Comment (m-doughty) [inline medicines/web/infrastructure/README.md]: Any example doc?

File: medicines/web/infrastructure/README.md (hunk 1 above)
Comment (rjkerrison) [reply medicines/web/infrastructure/README.md]: It has to be written. This documentation is to-be more than an as-is. I can perhaps make this clearer.

File: medicines/web/infrastructure/README.md (hunk 2 above)
Comment (rjkerrison) [reply medicines/web/infrastructure/README.md]: Best example we have would be the one for the API at `../../api/infrastucture/<environment>/deployment.yml`.
I can update to include this guidance.

Comment (m-doughty) [reply medicines/web/infrastructure/README.md]: Sounds good, or copy/paste it inline with values removed.

File: medicines/web/infrastructure/README.md (hunk 1 above)
Comment (m-doughty) [reply medicines/web/infrastructure/README.md]: Agreed.

**Pull request merged**